### DISCUSSION TITLE
Does addon-next also solve the next/font problem?

### DISCUSSION THREAD

**Discussion by Yedidya10** [RyanClementsHax/storybook-addon-next#175] [Q&A]
Is there a solution for [next font in version 13](https://nextjs.org/docs/api-reference/next/font#css-variables) to be displayed in storybook?

**Comment by RyanClementsHax** [comment]
As of now no, but I'd like to direct you to `@storybook/nextjs` anyway. I'm about to put this package in "keep the lights on mode" since the aforementioned package is the future of what this package was trying to accomplish. As such, I'm migrating everyone over to that package. If you run into any problems doing so, let us know by opening up an issue on storybook's side.
